docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,26 @@ weight: 35
 
 ## Unreleased
 
+## [v0.15.0](https://github.com/aws/eks-anywhere/releases/tag/v0.15.0)
+### Added
+- Kubernetes 1.26 support ([#5079](https://github.com/aws/eks-anywhere/issues/5079))
+- Workload clusters full lifecycle API support for Bare Metal provider ([#5237](https://github.com/aws/eks-anywhere/issues/5237))
+- Workload clusters full lifecycle API support for Nutanix provider
+- Support for InsecureSkipVerify for RegistryMirrorConfiguration across all providers ([#1647](https://github.com/aws/eks-anywhere/issues/1647))
+- Support for configuring of Bottlerocket settings. ([#707])(https://github.com/aws/eks-anywhere/issues/707)
+- Support for using a custom CNI ([#5217](https://github.com/aws/eks-anywhere/issues/5217))
```

Comment:
There's a gotcha we need to document and I think would be useful to either include here or link out to. How do we want to handle that? 

@chrisnegus

Reply:
@chrisdoherty4 I think it would be best to put a caveat at the point where the person would be using this feature. Where do you think that would be? Possibly in the getting started area or on clusterspec pages if the change would go there?

Reply:
That makes sense. The warning I've tried to include there covers the caveat.

Reply:
Confirming. So we don't need to add anything  here right?

Reply:
Yea the issue links to the Docs PR and the docs will have the warning in so no change required.